### ISSUE TITLE
Crash on restart

### ISSUE THREAD

**Issue by sx66627** [eggheads/eggdrop#1196]
Every time I enter restart command in partyline I get an error. 

Log:
```
[0:44:21] .restart
[00:44:19] #sx66627# restart
Restarting.
[00:44:19] Writing user file...
[00:44:19] Writing channel file...

[00:44:19] Restarting ...
[00:44:19] Module unloaded: console
[00:44:19] Module unloaded: notes
[00:44:19] Module unloaded: twitch
[00:44:19] Module unloaded: irc

[00:44:19] Module unloaded: ctcp

[00:44:19] Module unloaded: server
[00:44:19] Writing channel file...
[00:44:19] Module unloaded: channels
[00:44:19] Module unloaded: dns
[00:44:19] * Last context: twitch:twitch.c/812 []
[00:44:19] * Please REPORT this BUG!
[00:44:19] * Check doc/BUG-REPORT on how to do so.
[00:44:19] * Wrote DEBUG
[00:44:19] * SEGMENT VIOLATION -- CRASHING!

```

But it starts after without errors. 

DEBUG
```
Debug (eggdrop v1.9.1) written Thu Aug 26 00:44:19 2021
Patch level: stable
Tcl library: *unknown*
Tcl version: *unknown* (header version 8.6.8)
Tcl is threaded
Compiled with IPv6 support
Compiled with TLS support
Configure flags: none
Compile flags: gcc -g -O2 -pipe -Wall -I.. -I..  -DHAVE_CONFIG_H -I/usr/include/tcl8.6
Link flags: gcc
Strip flags: touch
Context: tclhash.c/729, []
         tclhash.c/734, [Tcl proc: <null>, param:  $_raw1 $_raw2 $_raw3]
         tclhash.c/749, []
         tclhash.c/729, []
         tclhash.c/734, [Tcl proc: *raw:irc:part, param:  $_raw1 $_raw2 $_raw3]
         tclhash.c/729, []
         tclhash.c/734, [Tcl proc: join:part, param:  $_p1 $_p2 $_p3 $_p4 $_p5]
         tclhash.c/749, []
         tclhash.c/749, []
         tclhash.c/729, []
         tclhash.c/734, [Tcl proc: <null>, param:  $_raw1 $_raw2 $_raw3]
         tclhash.c/749, []
         tclhash.c/729, []
         tclhash.c/734, [Tcl proc: *dcc:restart, param:  $_dcc1 $_dcc2 $_dcc3]
         tclhash.c/749, []
         twitch:twitch.c/812 []

IDX ADDR                                     + PORT NICK      TYPE  INFO
--- ---------------------------------------- ------ --------- ----- ---------
5   0.0.0.0                                    7530 (telnet)  lstn  7530
-1  0.0.0.0                                       0           lost
-1  0.0.0.0                                       0           lost
11  178.140.207.24                            58944 sx66627   chat  flags: cptEp/0

Compiled without extensive memory debugging (sorry).
Open sockets: 5 (listen), 6 (file), 11, done.
```

**Comment by vanosg** [comment]
Should be fixed in #1264 and available as part of 1.9.2 stable (coming VERY soon!)

**Comment by vanosg** [comment]
Closed by #1264
